api/src/main/java/gov/cms/ab2d/api/controller/BulkDataAccessAPI.java

```diff
@@ -1,31 +1,44 @@
 package gov.cms.ab2d.api.controller;
 
-
+import gov.cms.ab2d.api.service.JobService;
 import gov.cms.ab2d.api.util.Constants;
+import gov.cms.ab2d.domain.Job;
 import io.swagger.annotations.*;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.core.io.Resource;
 import org.springframework.http.HttpHeaders;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
 
 import javax.validation.constraints.NotBlank;
 import java.io.IOException;
+import java.util.Set;
+
+import static gov.cms.ab2d.api.util.Constants.API_PREFIX;
 
 @Api(value = "Bulk Data Access API", description =
         "API through which an authenticated and authorized PDP sponsor" +
                 " may request a bulk-data export from a server, receive status information " +
                 "regarding progress in the generation of the requested files, and retrieve these " +
                 "files")
 @RestController
-@RequestMapping(path = "/api/v1", produces = "application/json")
+@RequestMapping(path = API_PREFIX, produces = "application/json")
 /**
  * The sole REST controller for AB2D's implementation of the FHIR Bulk Data API specification.
  */
 public class BulkDataAccessAPI {
 
-    private static final String ALLOWABLE_OUTPUT_FORMATS =
-            "application/fhir+ndjson,application/ndjson,ndjson";
+    private static final Set<String> ALLOWABLE_OUTPUT_FORMAT_SET = Set.of("application/fhir+ndjson", "application/ndjson", "ndjson");
+
+    // Since this is used in an annotation, it can't be derived from the Set, otherwise it will be an error
```

Comment:
True. But you could derive the Set above from this constant to avoid duplication. Something like
```
Set.<String[]>of(ALLOWABLE_OUTPUT_FORMATS.split(","))
```